0_overview.md

```diff
@@ -66,3 +66,63 @@ Cowboy is an HTTP server written in Erlang by Loïc Hoguin of [99s](http://ninen
 Cowboy has fantastic documentation. The [Guides](http://ninenines.eu/docs/en/cowboy/HEAD/guide/) are especially helpful. Learning more about Cowboy will surely help you to understand Phoenix more fully.
 
 Cowboy has its own section of links in the [Resources Guide](http://www.phoenixframework.org/docs/resources).
+
+
+### Dependencies
+
+There are a number of dependencies external to Phoenix which we will encounter as we work our way through these guides. Elixir and Erlang are hard dependencies, meaning that we won't be able to work with Phoenix at all unless we have them installed. The rest may not prevent us from getting started if we don't have them, but their absence may lead to errors that prevent us from moving forward. Simply installing these dependencies may save us from confusion and frustration later on.
+
+Let's take a look at each of them now.
+
+- Elixir
+Phoenix is written in Elixir, and our application code will also be written in Elixir. In order to do any work with Phoenix, we need Elixir installed on our system. The Elixir site itself has great [installation instructions](http://elixir-lang.org/install.html).
+
+- Erlang
+Elixir source code compiles to Erlang byte code which runs on the Erlang Virtual Machine. That means we must have Erlang installed on our system - in addition to Elixir - in order to work with Phoenix. The Elixir site also has [Erlang installation instructions](http://elixir-lang.org/install.html#installing-erlang).
+
+- node.js
```

Comment:
I think we can be more straight-forward here. We are going into deep into bits like `--no-brunch` that don't make sense until we create the first app. Just make it clear when you'd need it and that you can easily skip it and we are good. Something like:

> Node is an optional dependency. If you want to work with assets (javascript, css, etc), by default Phoenix will use brunch.io for asset compilation. Brunch.io uses the node package manager (npm) to install its dependencies, and npm requires node.js.

> If you don't care about assets or you want to use another build tool, pass the `--no-brunch` when creating a new application and node won't be required at all.

Thoughts?